daemon/src/model/cfd.rs

```diff
@@ -658,6 +670,16 @@ impl Cfd {
             return Err(RolloverError::WrongRole);
         }
 
+        // TODO: Adjust for total paid fees
+        let margin_minus_total_fees = self.margin();
+
+        // TODO: Calculate the time from the last rollover, don't just assume
+        // it's up-to-date
+        let hours_to_charge = 1;
+
+        let funding_fee =
+            FundingFee::new(margin_minus_total_fees, self.funding_rate, hours_to_charge)?;
```

Comment:
Just pointing this out, we don't have to action it now, but we should keep in mind that this might not be what we want:

I am not sure collecting the fee based on our margin is correct. When discussing this with @bonomat we were talking about charging the fee on the `counterparty` margin, because that is what you borrow. 
That has the effect, that higher leverage will result in higher fees. 

Note that BitMEX is charging based on the amount of contracts (an abstraction that is still missing in our model):

> `Funding = Mark Value * Funding Rate`
> Your Mark Value is irrespective of leverage. For example, if you hold 100 XBTUSD contracts, funding is charged/received on the notional value of those contracts, and is not based on how much margin you have assigned to the position.

see: https://www.bitmex.com/app/perpetualContractsGuide#Funding

Maybe you can add a comment so we don't forget discussing this with @bonomat ?

Reply:
My apologies, I must've missed that. This reasoning makes sense. If you don't mind, instead of a comment, I will convert it to a separate issue - this would be relatively quick and independent fix if we decide to action it before tomorrow.

Reply:
sorry for slow reply. I've created a discussion for this topic: #1167